### ISSUE TITLE
Changes to Beaker description in screen summary (dynamic state of sim)

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#123]
After reviewing the dynamic state-of-sim in screen summary, I had three concerns. @Matthew-phet, @emily-phet and I discussed these concerns and possible solution in June 18th's design meeting.

I will list out the phrase changes here. The dynamic parameters should be the same as the ones defined in #115, #116 and #117.

The concerns were:
1. Repeat solute name, no mention of color. Decision: replace first mention of solute with the color property, instead!
2. Decision to remove a couple of extra small words "is" and "yet".
3. There was no mention of saturation when values checked? This was intentional, but we have decided to keep the saturated state to the summary without values checked.
# See update to phrasing in https://github.com/phetsims/molarity/issues/123#issuecomment-509726059
## Example strings should cover all cases:
For a side-by-side comparison of Old formulas and for more examples, please see the design document [Dynamic state-of-the-sim in Screen Summary](https://docs.google.com/document/d/1Z_5biEOBk9W4lVPXxGt7fyg-V6_CwTA4vMZ0mVMTJsI/edit#heading=h.t5o2rhu418i6)

### Case 1: Sim startup (normal case)
a. Values not checked: 
**Changed Phrase Formula:** 
Currently, beaker {{is volumeProperty (qualitative)}} of {{a color property}} solution containing {{soluteAmountProperty (qualitative)}} {{soluteProperty}} solute. Solution is {{concentrationProperty (qualitative)}} {{and state of saturation (don’t know property name for this)}}.

- **Change should result in:**
  - Currently, beaker {{is half full}} of {{a red ‬solution}} containing {{some}} {{‪drink mix‬}}‬ solute. Solution {{has low concentration}} {{and is not saturated}}.

b. Values checked: Note: Similar sentences, but with changed verb (has), and replace described values with actual measured values. Note: adding the saturation state to values checked phrase.
**Changed Phrase Formula:**
Currently, beaker {{has volumeProperty litres}} of {{a color property solution}} containing {{soluteAmountProperty moles of}} {{soluteProperty}} solute. Solution {{saturation state}} with concentration of {{concentrationProperty molar}}.

- **Change should result in:**
  - Currently, beaker {{has 0.500 liters}} of {{a red‬‬ ‬solution}} containing {{0.500 moles of}} ‪{{drink mix}}‬‬ solute. Solution {{is not saturated}} with concentration of {{1.000 molar}}.

### Case 2: When Solute is zero
a. Values not checked
**Changed Phrase Formula:**
Currently, beaker {{is volumeProperty}} of pure water with no {{soluteProperty}} solute.
- **Change should result in:** 
  - Currently, beaker {{is half full}} of pure water with no {{drink mix}} solute.

b. Values checked
**Changed Phrase Formula:**
Currently, beaker {{has volumeProperty litres}} of pure water with no {{soluteProperty}} solute.
- **Change should result in:** 
  - Currently, beaker {{has 0.500 litres}} of pure water with no {{drink mix}} solute.

**Comment by terracoda** [comment]
@Matthew-phet, can you review and discuss with @twant. I examples in the design doc, but I haven't copied all that information here. The only example not shown above is a saturated examples. Here's one Potassium Permanganate:

**Change should result in:**
a. Currently, beaker {{is over half full}} of {{a purple solution}} containing {{max amount of}} {{potassium permanganate}} solute. Solution {{{{has max concentration}}{{, and is saturated}}}}. 
b. Currently, beaker {{has 0.614 liters}} of {{a purple‬ ‬solution}} containing {{1.000 moles of}} ‪{{potassium permanganate}}‬‬ solute. Solution {{{{is saturated}}{{ with {{concentration of 0.500 molar}}}}.

**Comment by terracoda** [comment]
This issue is all yours, @twant. Let us know if you have questions.

**Comment by terracoda** [comment]
@Matthew-phet, I just need to verify the wording for the unit for solute amount "moles, mol, mols???" at our meeting today and then you can assign back to @twant.

I was reviewing #121 and I noticed I made some in consistent uses of the unit for solute amount.

**Comment by terracoda** [comment]
@twant, @Matthew-phet verified "moles" and "moles of" is correct. Please use these as needed.

**Comment by terracoda** [comment]
@twant, I think this is safe to hand back over to you. Please ask questions if you have any.

**Comment by twant** [comment]
@terracoda @Matthew-phet assigning back over to you for review

**Comment by terracoda** [comment]
@twant, we are almost there for there for when values are mot checked (qualitative descriptions); however, the phrases still need updating for the quantitative version when Solution Values is checked.

**When Solution Values NOT checked** 
- just remove "yet" from "is not yet saturated"

**When Solution Values checked**
- The new phrasing is not implemented. The requested saturation phrasing had sautration and concentration descriptions swapped. 
- that said, maybe @Matthew-phet and I can re-arrange our original request to make sure no order swapping is necessary between values checked and values not checked, regardless of whether the solution is saturated or not.

**Comment by terracoda** [comment]
@twant, heads up, regarding my previous comment, @Matthew-phet and I discussed some phrasing that would keep solution and concentration descriptions in the same order in all cases (no vales, values, saturated, and not saturated) in the dynamic screen summary.

I'll put the examples in the next comment...

**Comment by terracoda** [comment]
## Case 1: Sim startup (normal case)
### a. Values not checked: 
**Actual Example from A11y View July 9th**
> Currently, beaker is half full of a red ‬solution containing some ‪drink mix‬‬ solute. Solution is not concentrated and is not yet saturated.

**Requested Adjustment (July 9th):**
**Phrase Formula Qualitative**
**Note:** @twant, I bolded **is** and **has** in some places as I think they might just be part of the parameter. 
- Currently, beaker {{**is** volumeProperty}} of a {{colorProperty}} solution containing {{soluteAmountProperty}} {{soluteProperty}} solute. Solution is {{saturatedState}}, and {{concentrationProperty}}.

- **For unsaturated state the change should result in:**
    - Currently, beaker {{**is** half full}} of a {{red ‬solution}} containing {{some}} {{‪drink mix‬}}‬ solute. Solution is {{not saturated}} and {{**is** slightly concentrated}}.
- **For saturated state the change should result in:**
    - Currently, beaker {{**is** half full}} of a {{purple ‬solution}} containing {{some}} {{potassium‬ permanganate}}‬ solute. Solution is {{saturated}} and {{**has** max concentration}}.

### b. Values checked: 
**Note:** Similar sentences, but with changed verb has all the time before volume, replaces described values with actual measured values, adds **of** after "moles", and describes saturated state (is/is not) in all cases.

**Phrase Formula Quantitative:**
- Currently, beaker has {{volumeProperty}} litres of a {{colorProperty}} solution containing {{soluteAmountProperty}} moles of {{soluteProperty}} solute. Solution is {{saturatedState}}, and concentration is {{concentrationProperty}} molar.

- **For unsaturated state the change should result in:**
    - Currently, beaker has {{0.500}} liters of a {{red‬‬}} ‬solution containing {{0.500}} moles of ‪{{drink mix}}‬‬ solute. Solution is {{not saturated}}, and concentration is {{1.000 molar}}.
- **For saturated state the change should result in:**
    - Currently, beaker has {{0.500}} liters of a {{purple‬‬}} ‬solution containing {{0.500}} moles of ‪{{potassium‬ permanganate}} solute. Solution is {{saturated}}, and concentration is {{0.500 molar}}.

## Case 2: Edge cases when solute amount is at zero
### a. Values not checked: 
**Phrase Formula Qualitative:**
Currently, beaker {{**is** volumeProperty}} of pure water, and contains no {{soluteProperty}} solute.
- **Change should result in:** 
    - Currently, beaker {{**is** half full}} of pure water, and contains no {{drink mix}} solute.
    - Currently, beaker {{**is** half full}} of pure water, and contains no {{potassium permanganate}} solute.

## b. Values checked
**Phrase Formula Qualitative:**
Currently, beaker has {{volumeProperty}} litres of pure water, and contains no {{soluteProperty}} solute.
- **Change should result in:** 
    - Currently, beaker has {{0.500}} litres of pure water, and contains no {{drink mix}} solute.
    - Currently, beaker has {{0.500}} litres of pure water, and contains no {{potassium permanganate}} solute.

@twant, please notice the change to the phrasing when solution is pure water. We changed "with no" to ", and contains no".

**Comment by terracoda** [comment]
@twant, in the previous comment you find the changes that put the saturation and concentration descriptions in the same order for all cases, AND you find a slightly improved phrase for CASE 2 (zero solute).

I hope it is clear. Assigning back to you. You and @Matthew-phet can discuss at you meeting today.

**Comment by terracoda** [comment]
@twant, looks like Case 1b, Case 2a and 2b are implemented correctly, but Case 1a is not.

Could you review examples provided for Case 1a in https://github.com/phetsims/molarity/issues/123#issuecomment-509726059, and compare to your implementation. Saturation should come before concentration.

 This is what I see today for Case 1a:
**Unsaturated Solution**
<img width="634" alt="Screen Shot 2019-07-23 at 2 47 34 PM" src="https://user-images.githubusercontent.com/1902116/61713985-14d9f100-ad5a-11e9-854b-3e04b39fbc91.png">

**Saturated Solution**
<img width="629" alt="Screen Shot 2019-07-23 at 2 47 46 PM" src="https://user-images.githubusercontent.com/1902116/61713831-c4fb2a00-ad59-11e9-87c7-01edf3abaff1.png">

**Comment by terracoda** [comment]
@twant, I reviewed, so assigning back to you. We are almost there! Just one more correction.

**Comment by twant** [comment]
@terracoda @Matthew-phet pushed to master and will be in tonight's dev version. Assigning back over to you for review!

**Comment by terracoda** [comment]
@twant and @Matthew-phet, looks like Case 1a, with Solution Values not checked is perfect. And for Case 1a, Solution Values checked, we are just missing the final period. I took screen shots to demonstrate.

**Solution Values not checked**
<img width="686" alt="case_1a_dm_values_hidden" src="https://user-images.githubusercontent.com/1902116/62798442-1e21c800-ba93-11e9-9c34-27396781e853.png">
<img width="687" alt="case_1a_pp_values_hidden" src="https://user-images.githubusercontent.com/1902116/62798449-1feb8b80-ba93-11e9-90ea-d7cb6a1b6a99.png">

**Solution Values checked**
<img width="686" alt="case_1a_dm_values_checked" src="https://user-images.githubusercontent.com/1902116/62798484-342f8880-ba93-11e9-9664-0894368968a9.png">
<img width="687" alt="case_1a_pp_values_checked" src="https://user-images.githubusercontent.com/1902116/62798491-385ba600-ba93-11e9-87b4-7d51b10f7378.png">

@twant, once we have the period in, I think this issue is ready to close.
Nice work!

**Comment by twant** [comment]
@terracoda thanks for reviewing! I've added the period in and will push in a minute. Will prepare a dev release this evening for your review.

**Comment by terracoda** [comment]
@twant, I don't see the period on phetmarks after clearing caches. Are you sure you committed your changed?
The issue is when Solutions Values checked.
Please check again, verify and close issue when you get the period in there :-) Thanks!
<img width="901" alt="Screen Shot 2019-08-15 at 11 06 29 AM" src="https://user-images.githubusercontent.com/1902116/63115907-2ffce280-bf4d-11e9-93d4-0c54b32b2878.png">
<img width="905" alt="Screen Shot 2019-08-15 at 11 06 50 AM" src="https://user-images.githubusercontent.com/1902116/63115911-312e0f80-bf4d-11e9-9702-dd559d3059fc.png">

**Comment by terracoda** [comment]
@twant, I hope I am not missing something in my review process ;-)

**Comment by terracoda** [comment]
@twant, I poked around in the code to see where the period would go, but didn't find it, so leaving this for you :-)

**Comment by twant** [comment]
@terracoda apologies! I just checked and I had not pushed that commit after all. Just pushed it, and it should be fixed. now. Thanks for your patience with me!

**Comment by twant** [comment]
@terracoda Agh now the period is on the wrong one. Give me another couple of minutes to check and fix! will assign back to you when it's done.

**Comment by twant** [comment]
Okay, just pushed. Assigning back over to you!

**Comment by terracoda** [comment]
@twant, okay, I didn't see the period at first, but after I actually got all your changes and removed the one I made, it looks like we are good!
I also commented on my last commit here which removes the period I added.
https://github.com/phetsims/molarity/commit/d986050c36cf6adbf6b6ea5822c3156ad5232624

Thanks. I think I will close this issue finally.
